src/components/ecl-button/ecl-button.stories.ts

```diff
@@ -35,6 +38,7 @@ const getArgTypes = () => {
       name: 'Icon only',
       type: { name: 'boolean'},
       description: 'Hides the label (not for screen-readers)',
+      if: { arg: 'icon', truthy: true },
```

Comment:
very nice, can we make also the other controls related to the icon appear only when an icon has been selected..?

Reply:
added also icon transform and icon position